views/incubator/omniledger.tpl

```diff
@@ -0,0 +1,101 @@
+                 style="float: left; padding: 1em;"/>
+            There are other blockchains than Bitcoin, but they all solve the problem of
+            <strong>consensus</strong> for decentralized systems. Reaching a consensus between
+            different entities is not easy, as some of the entities can try to cheat.
+            So the conception of blockchains always takes into account that some of the
+            participants will try to cheat. If too many are cheating, the consensus will fail,
+            or invalid transactions might get accepted, leading to a failure of the
+            decentralization.
+        </td>
+    </tr>
+</table>
+
+<h2>OmniLedger</h2>
+
+<p>
+    When building blockchains, there are a number of technical challenges to overcome.
+    OmniLeger solves the following challenges:
+</p>
+<ul>
+    <li><strong>Speed</strong> - if many people want to use the system, it needs to be fast. Speed is measured
+        in two ways: transactions per second, and confirmation delay
+    </li>
+    <li><strong>Transactions per second</strong> - define how many money transfers can be processed by the system.
+        VISA can process up to 10'000 transactions in a second, while Bitcoin can do about 7.
+    </li>
+    <li><strong>Confirmation delay</strong> - how long you have to wait to be sure your money transfer arrived.
+        VISA takes about 5 seconds, Bitcoin takes about 1h.
+    </li>
+    <li><strong>Attack resistance</strong> - if you receive money in the system, you want to make sure it
+        stays there, and does not disappear all of a sudden.
+    </li>
+    <li><strong>Secret storage</strong> - today's blockchains are often world-readable, so it is important
+    that you still can store secret data without everybody being able to read it.</li>
+</ul>
+
+<h2>Links to demos</h2>
+
+<p>OmniLedger has already been used for a number of years, and there are a lot of projects that
+interact with this blockchain.
+Together with the <a href="https://dedis.ch">DEDIS</a> lab, C4DT is running a test-network for
+more than a year now, and it is used on a daily basis for different services.</p>
+
+<ul>
+    <li><a href="https://wookie.ch/columbus">Blockchain Explorer</a> -
+        as our test network is public, you can explore the blocks to see the latest transactions.</li>
+    <li><a href="https://status.dedis.ch">Node Status</a> -
+        here you can see the status of the nodes running in our test network.</li>
+    <li><a href="https://login.c4dt.org">Login Service</a> -
+        this service is used by C4DT and its partners as authentication service to the internal
+        pages. If you're interested, get in contact with <a href="mailto:linus.gasser@epfl.ch">Linus Gasser</a></li>
+</ul>
+
+<p>For more technical references like the paper describing OmniLedger, you can have a look at the
+<a href="https://factory.c4dt.org/showcase/labs/DEDIS/omniledger">showcase entry</a>.</p>
```

Comment:
probably we need to have a way to have it in each template, I still don't really know how to go from incubator to showcase

Reply:
My favorite would be a set of tags that can be used at the end to show
* papers
* code
* website
* all of the above

Perhaps something linked with the name of the description?

Reply:
My understanding [during our discussion](https://github.com/c4dt/showcase/issues/159#issuecomment-811090437) was that we did not want to duplicate the information here, and it is available in the main project page (i.e. on the showcase project-specific page).

In that spirit, having an (automated) link to the showcase project-specific page at the bottom of the presentation would be the simplest, wouldn't it?

Reply:
OK, to be discussed in another issue...

Reply:
#179